CI/Copyright-Checker/preserveCopyright.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * This file is part of ILIAS, a powerful learning management system
+ * published by ILIAS open source e-Learning e.V.
+ *
+ * ILIAS is licensed with the GPL-3.0,
+ * see https://www.gnu.org/licenses/gpl-3.0.en.html
+ * You should have received a copy of said license along with the
+ * source code, too.
+ *
+ * If this is not the case or you just want to try ILIAS, you'll find
+ * us at:
+ * https://www.ilias.de
+ * https://github.com/ILIAS-eLearning
+ *
+ * @author Thibeau Fuhrer <thibeau@sr.solutions>
+ */
+
+/**
+ * Function used by '@rollup/plugin-terser' to preserve only comments which
+ * contain a string indicating a copyright notice.
+ *
+ * Additionally, it only keeps ILIAS copyright notices if they are at the begining
+ * of a file, so that the license is not duplicated in the bundled file (this works
+ * due to plugins being ran after the bundle is created).
```

Comment:
No strong criticism here, just a thought: could other licenses than GPL-3.0-only appear in files which are included in a bundle? Is deduplication of the GPL-3.0-only reference still fine in these cases (→mix of licenses in bundle?)?

Just a question, because I feel that this _might_ become an issue, but you probably know better.